specification/search/data-plane/Azure.Search/stable/2023-11-01/searchindex.json

```diff
@@ -0,0 +1,2055 @@
+    },
+    "/docs/search.post.search": {
+      "post": {
+        "tags": [
+          "Documents"
+        ],
+        "operationId": "Documents_SearchPost",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/rest/api/searchservice/Search-Documents"
+        },
+        "x-ms-examples": {
+          "SearchIndexSearchDocumentsPost": {
+            "$ref": "./examples/SearchIndexSearchDocumentsPost.json"
+          }
+        },
+        "description": "Searches for documents in the index.",
+        "parameters": [
+          {
+            "name": "searchRequest",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SearchRequest",
+              "description": "The Search request."
+            },
+            "description": "The definition of the Search request."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ClientRequestIdParameter"
+          }
+        ],
+        "x-ms-request-id": "request-id",
+        "responses": {
+          "200": {
+            "description": "Response containing documents that match the search criteria.",
+            "schema": {
+              "$ref": "#/definitions/SearchDocumentsResult"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/SearchError"
+            }
+          }
+        }
+      }
+    },
+    "/docs('{key}')": {
+      "get": {
+        "tags": [
+          "Documents"
+        ],
+        "operationId": "Documents_Get",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/rest/api/searchservice/lookup-document"
+        },
+        "x-ms-examples": {
+          "SearchIndexGetDocument": {
+            "$ref": "./examples/SearchIndexGetDocument.json"
+          }
+        },
+        "description": "Retrieves a document from the index.",
+        "parameters": [
+          {
+            "name": "key",
+            "in": "path",
+            "required": true,
+            "description": "The key of the document to retrieve.",
+            "type": "string"
+          },
+          {
+            "name": "$select",
+            "in": "query",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "description": "List of field names to retrieve for the document; Any field not retrieved will be missing from the returned document.",
+            "x-ms-client-name": "SelectedFields"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ClientRequestIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response containing the requested document.",
+            "schema": {
```

Comment:
add LookupDocument definition then $ref it here? See https://github.com/Azure/azure-rest-api-specs/blob/main/specification/search/data-plane/Azure.Search/preview/2023-07-01-Preview/searchindex.json